src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -766,6 +771,7 @@ export const AutomatedAnalysisCard: React.FunctionComponent<AutomatedAnalysisCar
           'aria-expanded': isCardExpanded,
         }}
       >
+        <CardActions>{...props.actions || []}</CardActions>
```

Comment:
I guess we should do `...(props.actions || [])` since props.action can be `undefined`?

Reply:
I think that's what I wrote originally actually, but `yarn format:apply` seems to set it back to how it is now.

https://www.typescriptlang.org/play?ssl=1&ssc=1&pln=2&pc=1#code/PTAEHUFMBsGMHsC2lQBd5oBYoCoE8AHSAZVgCcBLA1UABWgEM8BzM+AVwDsATAGiwoBnUENANQAd0gAjQRVSQAUCEmYKsTKGYUAbpGF4OY0BoadYKdJMoL+gzAzIoz3UNEiPOofEVKVqAHSKymAAmkYI7NCuqGqcANag8ABmIjQUXrFOKBJMggBcISGgoAC0oACCbvCwDKgU8JkY7p7ehCTkVDQS2E6gnPCxGcwmZqDSTgzxxWWVoASMFmgYkAAeRJTInN3ymj4d-jSCeNsMq-wuoPaOltigAKoASgAywhK7SbGQZIIz5VWCFzSeCrZagNYbChbHaxUDcCjJZLfSDbExIAgUdxkUBIursJzCFJtXydajBBCcQQ0MwAUVWDEQC0gADVHBQGNJ3KAALygABEAAkYNAMOB4GRonzFBTBPB3AERcwABS0+mM9ysygc9wASnJjVl8sVSoCpoA2gBdHUAbmlBrlkAV8GVpoCXG4kGSGUgrgAPr7QJabcEVDhmh4yF5EOLnMD2DQvm4zMx2AxmJB+LBoOpEhz4HoRF4+XSGUzBHykti+eAHKgAOTCABykAkfKCKgA8l8yO9BBmtJAjqhHApXNI8KAnNGdMMsPoUAhENDhJdExJxdERAYONiFkxWBweEESkA

It seems that the TS compiler handles this in a typesafe way anyway, though it definitely doesn't look right reading it.

Reply:
Ahh interesting. Thanks for explaning. Agreed it was super odd XD